src/libcore/thread.cpp

```diff
@@ -85,12 +91,30 @@ class WorkerThread : public Thread {
 
 std::atomic<uint32_t> WorkerThread::m_counter{0};
 
+struct ThreadRegistrator {
+    ThreadRegistrator() {
+        // Do not register the main thread
+        if (m_counter > 0)
+            Thread::register_external_thread("worker");
```

Comment:
Give it a number? (with `snprintf("wrk_%02i", ...)`)

Reply:
This is just the "prefix", could change this to `wrk_` if you prefer that.